examples/ai21_examples/ai21_examples.ipynb

```diff
@@ -0,0 +1,342 @@
+    "import agentops"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we'll grab our API keys. You can use dotenv like below or however else you like to load environment variables"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "load_dotenv()\n",
+    "AI21_API_KEY = os.getenv(\"AI2I_API_KEY\") or \"<your_ai21_key>\"\n",
+    "AGENTOPS_API_KEY = os.getenv(\"AGENTOPS_API_KEY\") or \"<your_agentops_key>\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "agentops.init(AGENTOPS_API_KEY, default_tags=[\"ai21-example\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Setting up Messages\n",
+    "AI21 clients use a `ChatMessage` object to handle messages. We setup the following system prompt to guide the model in its response and a user prompt as well. We take the example of a support agent in a SaaS company."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "messages = [\n",
+    "    ChatMessage(\n",
+    "        content=\"You are a world renowned poet in the style of Edgar Allan Poe.\",\n",
+    "        role=\"system\",\n",
+    "    ),\n",
+    "    ChatMessage(\n",
+    "        content=\"Write me a short poem about the AI agents co-existing within the human brain.\",\n",
+    "        role=\"user\",\n",
+    "    ),\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sync Examples"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will demonstrate a basic sync call to AI21 using the Jamba 1.5 model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client = AI21Client(api_key=AI21_API_KEY)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "response = client.chat.completions.create(\n",
+    "    messages=messages,\n",
+    "    model=\"jamba-1.5-mini\",\n",
+    ")\n",
+    "print(response.choices[0].message.content)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following example shows how to record data from the streamed response using the Jamba 1.5 model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "response = \"\"\n",
+    "\n",
+    "stream_response = client.chat.completions.create(\n",
+    "    messages=messages,\n",
+    "    model=\"jamba-instruct\",\n",
+    "    stream=True,\n",
+    ")\n",
+    "\n",
+    "for chunk in stream_response:\n",
+    "    response += chunk.choices[0].delta.content\n",
```

Comment:
Need to wrap `chunk.choices[0].delta.content` in a `str(...)` because if the last chunk is None, it will error.

Reply:
Done